### ISSUE TITLE
I would like to add the program to explain Kadane's algorithm. Please assign this task to me @tanyagupta0201

### ISSUE THREAD

**Issue by sagnik2001** [tanyagupta0201/Hacktoberfest-2022#64]

**Comment by tanyagupta0201** [comment]
Assigned

**Comment by sagnik2001** [comment]
Created pull request #60  Kindly verify and merge

**Comment by MadhuSaini22** [comment]
@tanyagupta0201  Is this repo actually taking part in Hacktoberfest2022?

**Comment by MadhuSaini22** [comment]
@tanyagupta0201  Is this repo actually taking part in Hacktoberfest2022?
I mean, does merged PRs of this repo will be included in Hacktoberfest2022?

**Comment by tanyagupta0201** [comment]
> @tanyagupta0201 Is this repo actually taking part in Hacktoberfest2022? I mean, does merged PRs of this repo will be included in Hacktoberfest2022?

Yes, I have been adding this **hacktoberfest-accepted** label to all my merged Pull Requests. All the PRs merged by me are valid.
There was some other error which has been fixed now. Sorry for the inconvenience caused.

**Comment by sagnik2001** [comment]
Hey I have created a pr of this issue kindly review it
